examples/js/loaders/LDrawLoader.js

```diff
@@ -1375,6 +1381,8 @@ THREE.LDrawLoader = ( function () {
 			var bfcCull = true;
 			var type = '';
 
+			var currentConstructionStep = false;
```

Comment:
Would this variable better be called `completedConstructionStep` to indicate that the next subobject represents the beginning of a new "step"? It seems that `0 STEP` indicates the "end" of a construction step spec. Just a thought! I was trying to figure out exactly what this meant when reading through the code.

Reply:
It is more like a separator. For example, it can be optionally put at the start and at the end of the geometry of an object, and those two STEP are ignored.

The boolean flag on an object indicates that this object is the start of a step.

So instead of `completedConstructionStep`, what about `startingConstructionStep`? Or something similar.

Reply:
I've made a commit renaming to `startingConstructionStep`. Let me know.

Reply:
Looks good!